### ISSUE TITLE
document how to run/debug with a live copy of ICU and/or CLDR

### ISSUE THREAD

**Issue by markusicu** [unicode-org/unicodetools#210]
The Unicode Tools pull in snapshots of ICU and CLDR based on a Maven setting. Sometimes we need to test with the latest version, in particular when we make changes in ICU/CLDR and want to make sure we don't break the tools, or want to change the tools to adjust to such changes.

Please document how to change the Maven/Eclipse settings so that a live version of the dependency is used.

@echeran @srl295

**Comment by srl295** [comment]
@markusicu  I can take a look at this.
But please consider prioritizing https://unicode-org.atlassian.net/browse/ICU-21757 and https://unicode-org.atlassian.net/browse/ICU-21425 which is to say this would be simpler if ICU4J were itself fully mavenized.

**Comment by srl295** [comment]
related to instructions in #1800 ?

**Comment by echeran** [comment]
Yep, related.  Of course, the "twist" here is that both ICU and CLDR are deps for Unicode Tools, whereas in https://github.com/unicode-org/cldr/pull/1800 it was just ICU as a dep for CLDR.

I wonder, on top of that, would we want to set up a cronjob-style Github Action that runs those steps to be documented?  It would limit how long we could go without realizing that something went out of sync between Unicode Tools <-> {ICU, CLDR}.
